lib/rubocop/cop/rspec/shared_examples_style.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Enforces use of string to titleize shared examples.
+      #
+      # @example
+      #   # bad
+      #   it_behaves_like :foo_bar_baz
+      #   it_should_behave_like :foo_bar_baz
+      #   shared_examples :foo_bar_baz
+      #   shared_examples_for :foo_bar_baz
+      #   include_examples :foo_bar_baz
+      #
+      #   # good
+      #   it_behaves_like 'foo bar baz'
+      #   it_should_behave_like 'foo bar baz'
+      #   shared_examples 'foo bar baz'
+      #   shared_examples_for 'foo bar baz'
+      #   include_examples 'foo bar baz'
+      #
+      class SharedExamplesStyle < Cop
+        def_node_matcher :shared_examples_style, <<-PATTERN
+          (send nil? {#{(SharedGroups::ALL + Includes::ALL).node_pattern}} $sym)
+        PATTERN
+
+        def on_send(node)
+          shared_examples_style(node) do |ast_node|
+            checker = StringStyleChecker.new(ast_node)
+            return unless checker.offensive?
+
+            add_offense(checker.node, message: checker.message)
+          end
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            checker = StringStyleChecker.new(node)
+            corrector.replace(node.loc.expression, checker.preferred_style)
+          end
+        end
+
+        # :nodoc:
+        class StringStyleChecker
+          MSG = 'Prefer %<prefer>s over `%<current>s` ' \
+                'to titleize shared examples.'.freeze
+
+          attr_reader :node
+          def initialize(node)
+            @node = node
+          end
+
+          def offensive?
+            !node.str_type?
```

Comment:
This method will always return true, since `node` is (currently) always a string node. You can remove the `#offensive?` method altogether.